_episodes/05-plotting.md

```diff
@@ -114,3 +131,152 @@ plt.savefig('all_samples.png')
 >> {: .language-python}
 > {: .solution}
 {: .challenge}
+
+### Plotting with `x` and `y`
+
+The `plot` function creates a line plot. If there is only one argument in the function, it will use that as as the `y` variable, with the `x` variable just being a count.
+
+If we wanted to use the `Frame` column for our `x` value, we would add it as the first argument of the `plot` function. This won't change the graph at all.
+
+```
+import matplotlib.pyplot as plt
+
+plt.figure()
+plt.plot(data[:,0], data[:,1])
+```
+{: .language-python}
+
+However, if we did not want to plot every frame, or if the data we were plotting was not sequential, we would have to specify `x` and `y`.
+
+Let's try plotting every 100th frame. To do this we will use a new slicing syntax. To select frames at regular intervals, we can use the syntax `array[start:stop:interval]`.
+
+For example, to get every other row to the 10th row, we would use a start of `0`, and end of `10`, and an increment of `2`. We then use the `:` to get every column.
+
+~~~
+data[0:10:2, :]
+~~~
+{: .language-python}
+
+~~~
+array([[ 1.    ,  8.9542,  5.8024, 11.5478,  9.9557],
+       [ 3.    ,  9.0066,  6.0637, 13.0924, 11.3043],
+       [ 5.    ,  9.1294,  5.9365, 13.5321, 10.6279],
+       [ 7.    ,  8.8657,  5.9186, 12.9312, 10.2176],
+       [ 9.    ,  9.4184,  6.1993, 13.2614, 10.366 ]])
+~~~
+{: .output}
+
+If you want to go to the end, leave out the `end` variable, and just use two colons `::`.
+
+> ## Check your understanding
+> How would you get every 100th point in the first column?
+>> ## Answer
+>> ~~~
+>> data[0::100, 0]
+>> ~~~
+>> {: .language-python}
+>>
+>> We have used a `start` of 0, `end` to the end of the array, and an `increment` of `100`. We then got just the first column (column 0 - counting starts at 0!)
+> {: .solution}
+{: .challenge}
+
+Let's make a plot with every `100` points.
+
+~~~
+plt.figure()
+plt.plot(data[0::100,0], data[0::100,1])
+~~~
+{: .language-python}
+
+<img src="../fig/plotting_every100.png" title="Every 100 frames" style="display: block; margin: auto;" />
+
+### Customizing plot appearance
+We can change the marker or line type by adding another argument after our `x` and `y` values. For example, we can make our line dashed by adding `'--'`.
+
+~~~
+plt.figure()
+plt.plot(data[0::100,0], data[0::100,1], '--')
+~~~
+{: .language-python}
+
+<img src="../fig/plotting_every100dashed.png" title="Dashed line" style="display: block; margin: auto;" />
+
+Change the line color to a marker
+
+~~~
+plt.figure()
+plt.plot(data[0::100,0], data[0::100,1], 'o')
+~~~
```

Comment:
Ok, now that I see how you are using this.  I understand your notation above.  I still think you could give these slices variable names, but you don't really have to given this usage.